src/client/components/PayoutDetail.js

```diff
@@ -31,6 +32,37 @@ AmountWithLabel.defaultProps = {
   amount: 0,
 };
 
+const getBeneficiariesPercent = user => {
+  try {
+    const weight = parseFloat(_.get(user, 'weight', 0)) / 10000;
+    // eslint-disable-next-line react/style-prop-object
+    return <FormattedNumber value={weight} style="percent" />;
+  } catch (error) {
```

Comment:
In what kind of scenarios is it possible to make this code throw an error?

Reply:
Thought a string would make it throw an error but all it does is make it NaN, so i could remove that and just have a NaN check